pandas/core/arrays/categorical.py

```diff
@@ -2523,7 +2523,7 @@ def _factorize_from_iterable(values):
                                       ordered=values.ordered)
         codes = values.codes
     else:
-        cat = Categorical(values, ordered=True)
+        cat = Categorical(values, ordered=False)
```

Comment:
Interesting that this didn't cause any testing issues...

Reply:
Yes, it's striking. It's possible that that`multi.py` is essentially the only caller of `categorical._factorize_from_iterable`, and that the calls that `multi` makes happen to be such that within `Categorical`'s constructor, control is always routed so that the only time that `ordered` is used is to incorrectly raise the `TypeError` on lines 352-354. That's my best-guess explanation, at least.